src/discussions/common/AuthorLabel.jsx

```diff
@@ -65,7 +65,7 @@ function AuthorLabel({
       )}
       {authorLabelMessage && (
         <span
-          className={classNames('mr-3 font-size-14 font-style-normal font-family-inter font-weight-500', {
+          className={classNames('mr-1 font-size-14 font-style-normal font-family-inter font-weight-500', {
```

Comment:
This component is used in multiple components. Did you test all the components? Please also verify this change from Figma or all components.